README.md

```diff
@@ -1 +1,5 @@
-# Product_landing_trial
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please make the readme file in a professional way. So the reader of your code knows what you did and how you did it.

Reply:
## Hello @Iqbal-Elham 
Great work 😆  on the review 👍. You've done it. Your project is complete!

# Status Approved! 
## Congratulations!!! 🎉

## Highlights
- Use of GitHub inline commenting ✔️ 
- Paraphrased Suggestions and direct ✔️ 
- Comments are clear and easy to understand ✔️ 


![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f5061536e555a573145657970756c734f77642f67697068792e676966](https://user-images.githubusercontent.com/78151836/199058000-ba2a1274-f922-4a52-b859-799989aaa2b6.gif)


## Optional
- [ ] Please try as much as possible to use inline comments at all times as it provides context to your review and will point to the line or lines of code around the conversation.
- [ ] For the README file, you could point out sections of the file that needs to be worked on, it will give an expression of what needs to be done.
- [ ] You could use more emojis and gifs to give some encouragement